src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,43 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Arrays;
+import java.util.List;
+
 public class FileWork {
+    private static final char SPECIFIED_CHARACTER = 'w';
+
     public String[] readFromFile(String fileName) {
-        return null;
+        File file = new File(fileName);
+        StringBuilder stringBuilder = new StringBuilder();
+        List<String> strings;
+        String[] result;
+
+        try {
+            strings = Files.readAllLines(file.toPath());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read from FILE! " + fileName, e);
+        }
+
+        for (String word : strings.toString()
```

Comment:
```suggestion
           for (String line : strings) {
      String[] words = line.toLowerCase().split("\\W+");
      for (String word : words) {
        if (word.startsWith(SPECIFIED_CHARACTER)) {
          filteredWords.add(word);
        }
      }
    }
```
instead I suggest smt like this. that will make your code more readable (but declare regex as a constant)
filtereredWords - is the List<String> as well. You can sort it using Collections.sort() method and then convert to arr bu .toArray(new String[0]) method